core/.js/src/main/scala/io/udash/utils/FileUploader.scala

```diff
@@ -20,20 +20,26 @@ class FileUploader(url: Url) {
 
   /** Uploads provided `file` in a field named `fieldName` with optional additional request headers. */
   def uploadFile(
-    fieldName: String, file: File, extraData: Map[js.Any, js.Any] = Map.empty, additionalRequestHeaders: Map[RequestName, RequestValue] = Map.empty
+    fieldName: String, file: File, extraData: Map[String, String | Blob] = Map.empty, additionalRequestHeaders: Map[RequestName, RequestValue] = Map.empty
   ): ReadableModelProperty[FileUploadModel] =
     upload(fieldName, Seq(file), extraData = extraData, additionalRequestHeaders = additionalRequestHeaders)
 
   /** Uploads provided `files` in a field named `fieldName` with optional additional request headers. */
   def upload(
-    fieldName: String, files: Seq[File], extraData: Map[js.Any, js.Any] = Map.empty, additionalRequestHeaders: Map[RequestName, RequestValue] = Map.empty
+    fieldName: String, files: Seq[File], extraData: Map[String, String | Blob] = Map.empty, additionalRequestHeaders: Map[RequestName, RequestValue] = Map.empty
```

Comment:
Should we add possibility of adding blobName as well? According to provided doc default fileName may not be that helpful:
` The default filename for Blob objects is "blob".`

Reply:
This functionality was not previously exposed by the utility either and there is no matching support on the servlet side either. We can follow up with new feature when needed.